kola/tests/misc/nvidia.go

```diff
@@ -0,0 +1,52 @@
+package misc
+
+import (
+	"bytes"
+	"fmt"
+	"time"
+
+	"github.com/flatcar/mantle/kola/cluster"
+	"github.com/flatcar/mantle/kola/register"
+	"github.com/flatcar/mantle/util"
+)
+
+const (
+	CmdTimeout = time.Second * 300
+)
+
+func init() {
+	register.Register(&register.Test{
+		Name:        "cl.misc.nvidia",
+		Run:         verifyNvidiaInstallation,
+		ClusterSize: 1,
+		Distros:     []string{"cl"},
+		// This test is to test the NVIDIA installation, limited to AZURE for now
+		Platforms:     []string{"azure"},
+		Architectures: []string{"amd64"},
+	})
+}
+
+func verifyNvidiaInstallation(c cluster.TestCluster) {
+	m := c.Machines()[0]
```

Comment:
With
```suggestion
	if kola.AzureOptions.Size != "Standard_NC6s_v3" {
		c.Skip("skipping due to wrong instance size")
	}
	m := c.Machines()[0]
```
I think we could do this in scripts:
```
diff --git a/ci-automation/vendor-testing/azure.sh b/ci-automation/vendor-testing/azure.sh
index 17081b3598..c57210d651 100755
--- a/ci-automation/vendor-testing/azure.sh
+++ b/ci-automation/vendor-testing/azure.sh
@@ -74,7 +74,7 @@ query_kola_tests() {
     kola list --platform=azure --filter "${@}"
 }
 
-other_instance_types=()
+other_instance_types=('Standard_NC6s_v3')
 if [[ "${CIA_ARCH}" = 'amd64' ]]; then
     other_instance_types+=('V1')
 fi
@@ -85,6 +85,6 @@ run_kola_tests_on_instances \
     "${CIA_FIRST_RUN}" \
     "${other_instance_types[@]}" \
     '--' \
-    'cl.internet' \
+    'cl.internet' 'cl.misc.nvidia' \
     '--' \
     "${@}"
```